src/session.js

```diff
@@ -0,0 +1,22 @@
+function methods(request, opt) {
```

Comment:
Please add comments giving an overview of this module

Reply:
Added comments